ci-operator/step-registry/image-based-install-operator/reinstall/verify/image-based-install-operator-reinstall-verify-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install verify reinstall command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOF"
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }') 2>&1
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+mkdir -p /root/ibi-cluster
+cat > /root/ibi-cluster/kubeconfig <<-EOKUBECONFIG
```

Comment:
Add a comment saying what is this kubeconfig

Reply:
Done